common/src/main/java/org/jboss/gm/common/utils/GroovyUtils.java

```diff
@@ -90,7 +90,7 @@ public static void runCustomGroovyScript(Logger logger, InvocationStage targetSt
                 throw new ManipulationException("Mandatory annotation '@InvocationPoint(invocationPoint = ' not declared");
```

Comment:
Does this suffer from the same problem where ALL is being passed in not the actual stage point?

Reply:
Yes. I will fix it.

Reply:
I just changed it to pass in `targetStage` (stage it runs at) instead of `stage` (annotated).